CHANGELOG.md

```diff
@@ -3491,3 +3495,5 @@
 [@kenman345]: https://github.com/kenman345
 [@nijikon]: https://github.com/nijikon
 [@mikeyhew]: https://github.com/mikeyhew
+[@kenman345]: https://github.com/kenman345
```

Comment:
CI has failed. Would you please delete this line?
https://travis-ci.org/rubocop-hq/rubocop/jobs/410116964#L1119

```diff
 [@mikeyhew]: https://github.com/mikeyhew
-[@kenman345]: https://github.com/kenman345
 [@mkenyon]: https://github.com/mkenyon
```